Pipfile

```diff
@@ -6,10 +6,8 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-numpy = "*"
-scikit-learn = "*"
-pandas = "*"
-nltk = "*"
+eli5 = "==0.10.1"
+dill = "==0.3.2"
```

Comment:
Why do you have an exact version and not star? I would recommend changing to * so the package will be auto updated by depandabot.